extensions/wikia/Search/classes/Result.php

```diff
@@ -3,33 +3,38 @@
  * Class definition for Wikia\Search\Result
  */
 namespace Wikia\Search;
+
 use \Solarium_Document_ReadWrite as ReadWrite; #forward compatibility with v3
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)